README.md

```diff
@@ -19,7 +19,7 @@ local machine.
 Before the build user can be created, the following profile must exist in
 your AWS credentials file:
 
-* `cool-terraform-backend`
+- `cool-terraform-backend`
```

Comment:
This role's Terraform is setup to run against the CyHy account and not the COOL multi-account configuration. I believe this profile should be changed to something more appropriate as a result.

Reply:
See cisagov/skeleton-ansible-role-with-test-user#22.